src/plugins/telemetry/public/services/cloud.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export type { CloudSetup } from '../../../../../x-pack/plugins/cloud/public';
```

Comment:
Let's avoid doing that and just import `CloudSetup` explicitly from its owning plugin when and where it's necessary.